css/locationbar.css

```diff
@@ -17,12 +16,11 @@
 .progressbar {
   z-index: 99;
   display: block;
-  width: 500px;
-  height: 30px;
-  background-image: linear-gradient(110deg, var(--progressbar-color), var(--progressbar-color) 460px, transparent 460px);
-  background-size: 500px 30px;
+  width: 100%;
+  height: 3px;
+  margin-left: -100%;
   position: absolute;
-  top: 0;
+  top: 40px;
```

Comment:
Not a problem in practice because progress bar only exists when in-page progress bars do not. In other words, medium implements its own progress bar in order to _avoid_ triggering progress bar in the browser.